helm/kubescape/Chart.yaml

```diff
@@ -19,4 +19,4 @@ dependencies:
   - name: dependency_chart
     version: 0.0.0
   - name: kubescape-operator
-    version: 1.28.0
+    version: ">1.28.0"
```

Comment:
Is the lower bound for a particular reason or is this just to avoid bumping the requirement in the future? If it's the second, I'd be inclined to see if renovate could bump them together by adding a comment here so that the requirement version matches the chart version.